File: DeltaAPI/DeltaAPI/Models/HistoricoChamado.cs
using System.ComponentModel.DataAnnotations;

namespace DeltaAPI.Models
{
    public class HistoricoChamado
    {
        [Key]
        public int HistoricoId { get; set; }    // Chave primária explícita
        public int ChamadoId { get; set; }
        public DateTime DataEvento { get; set; }
        public string DescricaoEvento { get; set; }
        public int? UsuarioId { get; set; }
    }
}

File: DeltaApp/DeltaApp/app/src/main/java/br/unip/hermesbot/model/HistoricoChamado.kt
package br.unip.hermesbot.model

data class HistoricoChamado(
    val historicoId: Int? = null,
    val chamadoId: Int? = null,
    val dataEvento: String? = null,
    val descricaoEvento: String? = null,
    val usuarioId: Int? = null
)
